static/docs/get-started/connect-code-and-data.md

```diff
@@ -48,17 +48,19 @@ recommend creating a virtual environment with a tool such as
 $ virtualenv -p python3 .env
 $ source .env/bin/activate
 $ echo ".env/" >> .gitignore
-$ pip install -r requirements.txt
+$ pip install -r src/requirements.txt
 ```
 
+</details>
+
 Save the progress to Git:
 
 ```dvc
 $ git add .
-$ git commit -m "add code"
+$ git commit -m "Add source code files to repo"
 ```
 
-</details>
+## Create a first data transformation stage
```

Comment:
I would avoid using second level titles. If there is a temptation to use it it means that section has become too complicated. The ideal size of the chapter for the get started - one screen. It should be simple, just enough to give some sense of the idea.

Reply:
Updated in 60a998b per chat IRL.